src/ScriptCs.Core/Repl.cs

```diff
@@ -192,12 +192,52 @@ private void HandlePaketReferences(IEnumerable<string> references)
         {
             var paketRefs = references.Where(r => r.StartsWith(Constants.PaketPrefix)).Select(r => r.Substring(Constants.PaketPrefix.Length)).ToArray();
             var refs = Path.Combine(FileSystem.CurrentDirectory, "Refs.csx");
-            var fi = new FileInfo(refs);
-            dynamic result = PaketShim.ResolveLoadScript(fi, paketRefs, s =>
+            
+            // todo: is this the .csx we are currently evaluating?
+            var scriptFileBeingProcessed = new FileInfo(refs);
```

Comment:
no, refs is not that file. In the REPL there may not even be a file. It depends. If you load scriptcs with -r you can specify a file for it to load the REPL with, but this is more of an edge case. I'd say in the REPL we just use the current directory as the hash and we can use a fixed filename like "repl.csx"